packages/experimental-utils/src/ts-eslint/Linter.ts

```diff
@@ -119,6 +119,14 @@ namespace Linter {
   export type RuleEntry = RuleLevel | RuleLevelAndOptions;
   export type RulesRecord = Partial<Record<string, RuleEntry>>;
 
+  export type GlobalVariableOption =
+    | 'readonly'
+    | 'writable'
+    | 'readable'
+    | 'writeable'
+    | 'off'
+    | boolean;
```

Comment:
because these are considered deprecated - let's leave them out.
For the types where possible we've been excluding deprecated values/apis to help push people towards the correct things and prevent them from using old things that will be removed in a future version.

In the next major we can consider removing the `boolean` as well, considering that is deprecated and instead the string values are preferred.

```suggestion
    | 'readonly'
    | 'writable'
    | 'off'
    | boolean;
```

Reply:
It looks better:+1: I fixed it!